scrapers/benefitmonkey.yml

```diff
@@ -0,0 +1,31 @@
+name: BenefitMonkey
+sceneByURL:
+  - action: scrapeJson
+    url:
+      - benefitmonkey.com/scenes/
+      - lezkey.com/scenes/
+    queryURL: https://benefitmonkey.com/_next/data/c0nOrXwXZNfyZr_FH-gFq/scenes/{url}.json

```

Comment:
```c0nOrXwXZNfyZr_FH-gFq``` needs to be dynamic, the ID changes for each scene.

It would probably be easier to use an XPath scraper and scan the scene url, the json URL doesn't seem to be predictable